app/ExceptionExfiltrator.m

```diff
@@ -0,0 +1,290 @@
+//
+//  ExceptionExfiltrator.m
+//  libiSHApp
+//
+//  Created by Saagar Jha on 5/5/23.
+//
+
+#import "ExceptionExfiltrator.h"
+#import <Foundation/Foundation.h>
+#import <execinfo.h>
+#import <pthread.h>
+
+#define f(name, character)         \
+	__asm__("\" " name "\": nop"); \
+	void ish_exception_exfiltrate_##character(void) __asm__(" " name)
+
+f("unprintable", unprintable);
+
+f(" ", space);
```

Comment:
Can we use X macros to fix the duplication

Reply:
Probably not